packages/application-config/schema.json

```diff
@@ -148,6 +155,10 @@
         "permissionsPolicies": {
           "description": "Configuration for the HTTP Permissions-Policy header (https://github.com/w3c/webappsec-permissions-policy/blob/main/permissions-policy-explainer.md)",
           "type": "object"
+        },
+        "strictTransportSecurity": {
+          "description": "Additional configuration for the HTTP Strict-Transport-Security header (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)",
+          "$ref": "#/definitions/hstsDirective"
```

Comment:
Nit: a ref is only useful if you reuse the object within the schema. In this case you could just "inline" it.

Reply:
Not sure I recall 100% but inlining it might have produced different typings.